### ISSUE TITLE
Nethunter on One Plus One(CM 11) Struck in Bootloop

### ISSUE THREAD

**Issue by citymonk** [offensive-security/kali-nethunter#241]
After Successfully flashing Nethunter on One Plus One(CM 11) and rebooting , the Phone is struck in Bootloop.It keeps on restarting after showing 1+ Logo.

I tried to go into TWRP and rebooted the phone , but it still doesn't move past the 1+ Logo.

Steps followed for Installation :

1.Unlocked OPO
2.Installed TWRP recovery
3.Rooted OPO
4.Installed Busybox
5.Flashed Nethunter for OPO (CM11)

The System showed the install was complete with out any errors.

I tried flashing 3 times but still didn't help.Cleared the Cache /Dalvik still not resolved.

Please suggest if I missed anything.

**Comment by binkybear** [comment]
@citymonk - I believe the latest update has broken the kernel.  More specifically, I included a ramdisk in the last kernel which probably isn't playing well with new update.

I will be test a a kernel today and if you are willing to flash I will upload it here.

**Comment by citymonk** [comment]
Hi @binkybear Thanks for the response.After trying multiple solutions from various sites and mixing them , the Nethunter was flashed properly.

The steps I followed :
1.I flashed the YOG4PAS1N0 64GB image by following the steps from this link  http://forum.xda-developers.com/oneplus-one/general/guide-return-opo-to-100-stock-t2826541. This will install CM12 on the mobile.
2.Used Windows Installer from http://www.nethunter.com/download/  till the step to flash the stock rom.(This will downgrade the system to CM11)
3.Downloaded the Kali Nethunter zip file for CM11 , from the Nethunter download page https://www.offensive-security.com/kali-linux-nethunter-download/
5.The install was successful after reboot.

ps : Also found that upgrading too CM12 after Nethunter installation , and also to 12.1 didn't give me any issues.
